File: myfancypdfinvoices/src/main/java/sk/janmokry/context/Application.java
package sk.janmokry.context;

import com.fasterxml.jackson.databind.ObjectMapper;
import sk.janmokry.service.InvoiceService;
import sk.janmokry.service.UserService;

public class Application {

    public static final UserService userService = new UserService();
    public static final InvoiceService invoiceService = new InvoiceService(userService);
    public static final ObjectMapper objectMapper = new ObjectMapper();
}

File: 01 exercise/mybank/src/main/kotlin/sk/janmokry/context/Application.kt
package sk.janmokry.context

import com.fasterxml.jackson.databind.ObjectMapper
import com.fasterxml.jackson.databind.SerializationFeature
import sk.janmokry.service.TransactionService

object Application {
    val transactionService by lazy { TransactionService() }
    val jacksonObjectMapper by lazy {
        ObjectMapper().apply {
            disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
        }
    }
}